notebooks/Tutorial_Top-K_Multidimensional_Motif_and_Matches_Discovery.ipynb

```diff
@@ -0,0 +1,827 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    from stumpy.mmotifs import mmotifs
</pre><p>We don't need this line anymore. Instead, with the latest version of <code>stumpy</code>, you should be able to call <code>stumpy.mmotifs</code> directly   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>